integrations/tensorflow/bindings/python/pyiree/tf/support/tf_utils.py

```diff
@@ -280,6 +320,33 @@ def create_from_instance(cls,
     """
     raise NotImplementedError()
 
+  @classmethod
+  def create_from_signature_def_saved_model(cls,
+                                            saved_model_dir: str,
+                                            saved_model_tags: Set[str],
+                                            module_name: str,
+                                            backend_info: "BackendInfo",
+                                            exported_name: str,
+                                            input_names: Sequence[str],
+                                            output_names: Sequence[str],
+                                            artifacts_dir: str = None):
```

Comment:
do we need test/doc updates for the artifacts_dir in the SignatureDef SM cases?

Reply:
Done!